build.sbt

```diff
@@ -1,54 +1,16 @@
 import sbt._
 
-Global / useGpg := false
+Universal / javaOptions ++= Seq("-Xms1g", "-Xmx2g", "-Xss512m", "-XX:+UseG1GC", "-XX:+UseStringDeduplication")
 
-// Sonatype repository settings
-val sonatypePublishSettings = Seq(
-  publishMavenStyle := true,
-  Test / publishArtifact := false,
-  Docker / publish := {},
-  Docker / publishLocal := {},
-  pomIncludeRepository := (_ => false),
-  publishTo := sonatypePublishToBundle.value)
-
-val sonatypeInformation =
-  Seq(
-    organizationName := "Codacy",
-    organizationHomepage := Some(new URL("https://www.codacy.com")),
-    startYear := Some(2018),
-    licenses := Seq("AGPL-3.0" -> url("https://opensource.org/licenses/AGPL-3.0")),
```

Comment:
Shouldn't we keep the licenses here?

Reply:
They are already defined in the Codacy Sbt Plugin